doc/plugins.dox

```diff
@@ -444,30 +535,170 @@ native spelling inside parenthesis, where applicable.
 
 You can (and should) also mark other strings beside the plugin's meta
 information as translatable.  Strings used in menu entries, information
-boxes or configuration dialogs should also be translatable as well.  Geany
-offers a way to enable this in the plugin's code using the main_locale_init()
-function provided by the plugin API. This function takes the same two
-parameters discussed in the previous section; @a GETTEXT_PACKAGE and
-@a LOCALEDIR.
-
-The main_locale_init() function is best called during initialization in the
-plugin's plugin_init() function.  Adding this to the HelloWorld example could
-look like:
+boxes or configuration dialogs should also be translatable as well. To enable this
+use main_locale_init() inside @a funcs->init, if you haven't done already in
+@a geany_load_module()
```

Comment:
IMO this shouldn't be advertized.  `main_locale_init()` should always be called before filling the infos, and infos always translated.  Adding this alternative here will only confuse some people and possibly lead to missing translations of plugin infos.